lib/src/analyzers/lint_analyzer/rules/rules_list/prefer_provide_intl_description/prefer_provide_intl_description_rule.dart

```diff
@@ -0,0 +1,46 @@
+// ignore_for_file: public_member_api_docs
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+
+import '../../../../../utils/node_utils.dart';
+import '../../../lint_utils.dart';
+import '../../../models/internal_resolved_unit_result.dart';
+import '../../../models/issue.dart';
+import '../../../models/severity.dart';
+import '../../models/intl_rule.dart';
+import '../../rule_utils.dart';
+
+part 'visitor.dart';
+
+class PreferProvideIntlDescriptionRule extends IntlRule {
+  static const String ruleId = 'prefer-provide-intl-description';
+
+  static const _warning = 'Prefer provide description for translated message';
```

Comment:
`'Provide description for translated message.';`

Reply:
fixed